### PR TITLE
Replace TCE font icons with SVGs

### PR DESCRIPTION
**Pull request by QueerSynd**
# What are the relevant GitHub issues?

resolves [#1060](https://github.com/crimethinc/website/issues/1060)

# What does this pull request do?

The font icons in the top navigation of the TCE landing page have been replaced by svgs. They also have been resized to keep the alignment with their labels.

# How should this be manually tested?

Visit the TCE page, check that the icons in the top bar are properly labeled and coloured, and that alignment with the labels is respected on various screen sizes.

# Screenshots
## Mobile
localhost_3000_tce
## Tablet
crimethinc com_tce (2)
## Laptop
crimethinc com_tce (1)
## Desktop
crimethinc com_tce

### CODE REVIEW

**Comment by just1602** [comment]
Can you tell what broser and operating system you're using?

I'm pretty sure that's something on my setup that doesn't work. But for me on firefox developer edition on linux, the blue icon on the left of the share menu doesn't show up. On chromium on linux it's appearing but the resize doesn't seems to be apply.

But I just wanna know to test it, even if I'm pretty sure it's just my setup which is missing some fonts or stuff liek that.

**Comment by QueerSynd** [comment]
Hi @just1602 
I took the screenshots on Chrome and OSX. A bit weird in my opinion that an asset would show up in a browser and not another, it is even loading?

**Comment by just1602** [comment]
Oh sorry for wasting your time. You're right, it's ublock origin that is blocking it. I don't know why, but yeah. I'll let @veganstraightedge validate it to be sure, but it sounds good to me. :)

File: app/assets/stylesheets/to_change_everything/_common.scss [hunk 1]
```diff
@@ -12,1062 +12,1072 @@
 
 /*------------------MORE TEXTS LINKS--------------------------*/
 
+/*------------------FOOTER----------------------------------*/
+
+/*------------------RESPONSIVE VIDEO EMBED------------------*/
```
Comment (veganstraightedge) [inline app/assets/stylesheets/to_change_everything/_common.scss:17]: Are all of the changes in this file related to TCE SVG icons?
Or is there a rebase missing?
Can we move them to a separate PR if they're not part of the TCE SVG icons change, please?

Comment (veganstraightedge) [reply app/assets/stylesheets/to_change_everything/_common.scss:17]: Ooooh. Is this just a indentation/linting diff?
If so, for PR review readability and clarity and stuff, would you mind breaking all of the linter/style changes out into a separate PR, please?

Then this PR can be really narrowly scoped and focused.

Comment (QueerSynd) [reply app/assets/stylesheets/to_change_everything/_common.scss]: I reverted the linting changes. I believe _common.scss still indicates changes because of the comments here, don't mind this file, there aren't any changes relevant in it.
I will create another PR for linting changes after this has been merged to avoid conflicts.

**Review by veganstraightedge — APPROVED** [review]
Looks good. Ship it! :shipit:
